packages/react-native/src/Authenticator/Defaults/ConfirmResetPassword/ConfirmResetPassword.tsx

```diff
@@ -41,23 +37,26 @@ const ConfirmResetPassword: DefaultConfirmResetPasswordComponent = ({
     handleSubmit,
   });
 
+  const headerText = getResetYourPasswordText();
+  const primaryButtonText = isPending ? getSubmittingText() : getSubmitText();
+  const secondaryButtonText = getResendCodeText();
+
+  const buttons = useMemo(
+    () => ({
+      primary: { children: primaryButtonText, onPress: handleFormSubmit },
+      secondary: { children: secondaryButtonText, onPress: resendCode },
```

Comment:
Should the `Resend Code` Button  be `type='link'`?

![Screen Shot 2022-11-21 at 11 39 41 AM](https://user-images.githubusercontent.com/48109584/203123530-da46d245-7c2b-4a8c-9da7-d3b70da78feb.png)

Reply:
Just addressed feedback form @calebpollman and changed from link to secondary. do we need to all sync? :)